src/main/java/org/dependencytrack/tasks/repositories/ComposerMetaAnalyzer.java

```diff
@@ -46,9 +53,32 @@ public class ComposerMetaAnalyzer extends AbstractMetaAnalyzer {
     private static final String DEFAULT_BASE_URL = "https://repo.packagist.org";
 
     /**
-     * @see <a href="https://packagist.org/apidoc#get-package-metadata-v1">Packagist's API doc for "Getting package data - Using the Composer v1 metadata (DEPRECATED)"</a>
+     * @see <a href="https://packagist.org/apidoc#get-package-data">Packagist's API doc for "Getting package data - Using the Composer v1 metadata (DEPRECATED)"</a>
+     * Example: https://repo.packagist.org/p/monolog/monolog.json
+     */
+    private static final String PACKAGE_META_DATA_PATH_PATTERN_V1 = "/p/%package%.json";
+
+    private static final Cache<String, JSONObject> REPO_ROOT_CACHE = Caffeine.newBuilder()
+                                                                    .expireAfterWrite(10, TimeUnit.MINUTES)
```

Comment:
Are we expecting this data to change very frequently? I'd think a cache duration of ~1h should still be ok, WDYT?

Reply:
That was my feeling as well. For now I just used 10 minutes which is what Composer CLI does.

Reply:
Changed it to 1 hour.